app/assets/stylesheets/components/_layout.scss

```diff
@@ -43,26 +43,35 @@
   }
 }
 
-.homepage-hero {
-  margin-bottom: $site-margins;
+.usa-hero {
+  background-image: url(asset_path("homepage/hero.svg"));
+  background-position: left center;
+  height: 173px;
```

Comment:
What's our policy on using px in CSS? I imagine that it's appropriate sometimes, but is this one of those times?

Reply:
I'll switch these to rem